pkg/cloudfabric-controller/controllerframework/controllerinstancemanager.go

```diff
@@ -120,10 +120,10 @@ func GetInstanceId() types.UID {
 	}
```

Comment:
Do we really need this changes here in this file ?

Reply:
> Do we really need this changes here in this file ?

Necessary for compilation